storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -338,6 +345,27 @@ private MesosSchedulerDriver createMesosDriver() throws IOException {
     return driver;
   }
 
+  private boolean isAccepted(Offer offer) {
```

Comment:
Naming suggestions for clarity:
* `isAccepted` -> `isOfferAccepted`
* `isAttributeAccepted` -> `isOfferAttributesAccepted`
* `isHostAccepted` -> `isOfferHostAccepted`  (to be consistent with the other naming suggestions)
* `attributes` -> `offerAttrList`
* `attributeMap` -> `offerAttrMap`
* `attr` -> `offerAttr`
* `entry` -> `constraint`
* `value` -> `offerAttrValue`